VitalsSimplification/BPMCheker.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace VitalsSimplification
+{
+    class BPMCheker
+    {

```

Comment:
This class is very similar to other checkers (like RespRateChecker.cs). Try having a single RangeChecker and using it with different limits for different vitals.